### DISCUSSION TITLE
Excellent work!

### DISCUSSION THREAD

**Discussion by alanv72** [PJDude/dude#16] [General]
Just wanted to drop a thank you. This was worked perfectly on Windows 11 workstation. I am assimilating many family members' multimedia into a big private repository. This tool is extremely useful!

**Comment by PJDude** [comment]
Thank you for your kind words, I really appreciate it.

**Reply by alanv72** [reply]
No problem! One feature request is to add a scan filter based on min file size.

**Reply by PJDude** [reply]
try latest pre-release:
https://github.com/PJDude/dude/releases/tag/v2.0219.473
